docs/reference/api-conventions.asciidoc

```diff
@@ -214,6 +214,37 @@ instead. When using this method, the `source_content_type` parameter
 should also be passed with a media type value that indicates the format
 of the source, such as `application/json`.
 
+[discrete]
+[[api-compatibility]]
+=== REST API version compatibility
+
+Major version upgrades often include a number of breaking changes 
+that impact how you interact with {es}. 
+While we recommend that you monitor the deprecation log and 
+update applications before upgrading {es},
+having to coordinate the necessary changes can be an impediment to upgrading. 
```

Comment:
```suggestion
implementing the necessary changes can be an impediment to upgrading. 
```

Reply:
"having to coordinate" feels imprecise, thought that might be the intention given that we're talking about all the things that need to happen for an upgrade.

Reply:
It was deliberate. Using the compatibility headers enables you to decouple the implementation and deployment of app updates from the upgrade process.